hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/SystemFileUpdateFacilityTest.java

```diff
@@ -32,18 +33,23 @@
 import com.hedera.node.app.config.VersionedConfigImpl;
 import com.hedera.node.app.fees.ExchangeRateManager;
 import com.hedera.node.app.fixtures.state.FakeHederaState;
+import com.hedera.node.app.hapi.utils.sysfiles.domain.throttling.ThrottleGroup;
 import com.hedera.node.app.service.file.FileService;
 import com.hedera.node.app.spi.fixtures.TransactionFactory;
 import com.hedera.node.app.throttle.ThrottleManager;
+import com.hedera.node.app.workflows.handle.record.SingleTransactionRecordBuilderImpl;
 import com.hedera.node.config.converter.BytesConverter;
 import com.hedera.node.config.converter.LongPairConverter;
 import com.hedera.node.config.data.FilesConfig;
 import com.hedera.node.config.data.LedgerConfig;
 import com.hedera.pbj.runtime.io.buffer.Bytes;
+import com.hederahashgraph.api.proto.java.HederaFunctionality;
 import com.swirlds.test.framework.config.TestConfigBuilder;
+import java.time.Instant;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
-import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Assertions;
```

Comment:
We should favor AssertJ instead of JUnit Jupiter assertions

Reply:
I didn't know that we were doing that. Fixed 👍